src/ReactHTMLElement.ts

```diff
@@ -50,15 +55,27 @@ class ReactHTMLElement extends HTMLElement {
     }
   }
 
+  get retargetCleanup(): () => void {
+    return this.retargetCleanupFunction;
+  }
+
+  set retargetCleanup(cleanupFunction: () => void) {
+    // Ensure that we cleanup an old listeners before we forget the cleanup function.
+    this.retargetCleanup();
+    this.retargetCleanupFunction = cleanupFunction;
+  }
```

Comment:
We still need to run this setter when we call `retargetReactEvents`.

Reply:
I thought I had done that. I guess I thought about it too much, but never did add the setters. Okay so this should work now.